app/code/Magebit/Faq1/Api/Data/QuestionInterface.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magebit\Faq1\Api\Data;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+
+/**
+ * Interface that defines all variables required getters and setters
+ */
+interface QuestionInterface extends ExtensibleDataInterface
+{
+    const QUESTION_ID = 'id';
+    const QUESTION = 'question';
+    const ANSWER = 'answer';
+    const STATUS = 'status';
+    const POSITION = 'position';
+    const UPDATED_AT = 'updated_at';
+
+    /**
+     * Get ID
+     *
+     * @return int|null
+     */
+    public function getId(): ?int;
+
+    /**
+     * Get Question
+     *
+     * @return string
+     */
+    public function getQuestion(): string;
```

Comment:
Add ?string here as well and also for the rest of get methods as there may be possibility that data is missing